cmd/xcode.go

```diff
@@ -81,12 +82,23 @@ func scanXcodeProject(cmd *cobra.Command, args []string) error {
 
 	projectPath := paramXcodeProjectFilePath
 	if projectPath == "" {
-		askText := `Please drag-and-drop your Xcode Project (` + colorstring.Green(".xcodeproj") + `) or Workspace (` + colorstring.Green(".xcworkspace") + `) file, 
+
+		log.Infof("Scan the directory for project files")
+		projpth, err := scanForProjectFiles([]ProjectType{xcodeWorkspace, xcodeProject})
```

Comment:
what if we find 1 solution, but unfortunately not the one the user wants to use? the logic does not fail, so the user has no option to use another solution file